### ISSUE TITLE
Goals

### ISSUE THREAD

**Issue by mmstick** [MakotoE/rust-fahapi#27]
Do you have any particular goals for this project? Any GUI frontend clients you're working on with this?

**Comment by MakotoE** [comment]
No, it's in maintenance mode. I did make [rust-fah-pauser](https://github.com/MakotoE/rust-fah-pauser) which I always run in the background, and it just pauses FAH when I'm running an IDE so that it doesn't get slowed down.
